src/ProjectFileIO.cpp

```diff
@@ -648,7 +782,29 @@ bool ProjectFileIO::GetValue(const char *sql, wxString &result)
       return 1;
    };
 
-   return Query(sql, cb);
+   return Query(sql, cb, silent);
+}
+
+bool ProjectFileIO::GetValue(const char *sql, int64_t &value, bool silent)
+{
+   bool success = false;
+   auto cb = [&value, &success](int cols, char** vals, char**)
+   {
+      if (cols > 0)
+      {
+         const std::string_view valueString = vals[0];
+
+         success = std::errc() ==
+            FromChars(
+               valueString.data(), valueString.data() + valueString.length(),
+               value)
+               .ec;
```

Comment:
Is it OK to ignore .ptr in the FromCharsResult?  (This is the only place you do ignore it)